web/src/app/alerts/CreateAlertDialog/DialogNavigation.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import { DialogActions, Button } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = makeStyles(theme => ({
+  button: {
+    marginRight: theme.spacing(1),
+  },
+  dialogActions: {
+    display: 'flex',
+    flexDirection: 'row-reverse',
+    justifyContent: 'flex-start',
+  },
+}))
+
+const nextIsDisabled = (activeStep, formFields) => {
+  switch (activeStep) {
+    case 0:
+      return !(formFields.summary && formFields.details)
+    case 1:
+      return formFields.selectedServices.length === 0
+    default:
+      return false
+  }
+}
+
+export default props => {
+  const {
+    activeStep,
+    formFields,
+    onClose,
+    onLastStep,
+    onSubmit,
+    setActiveStep,
+    steps,
+  } = props
+  const classes = useStyles()
+
+  const stepForward = () => {
+    setActiveStep(prevActiveStep => prevActiveStep + 1)
+  }
+
+  const stepBackward = () => {
+    setActiveStep(prevActiveStep => prevActiveStep - 1)
+  }
+
+  const getNextBtnLabel = () => {
+    switch (activeStep) {
+      case steps.length - 1:
+        return 'Done'
+      case steps.length - 2:
+        return 'Submit'
+      default:
+        return 'Next'
+    }
+  }
+
+  const getBackBtnLabel = () => {
+    switch (activeStep) {
+      case 0:
+        return 'Cancel'
+      default:
+        return 'Back'
+    }
+  }
+
+  const handleNext = () => {
+    switch (activeStep) {
+      case steps.length - 1:
+        onClose()
+        break
+      case steps.length - 2:
+        onSubmit()
+        stepForward()
+        break
+      default:
+        stepForward()
+    }
+  }
+
+  const handleBack = () => {
+    switch (activeStep) {
+      case 0:
+        return onClose()
+      default:
+        return stepBackward()
+    }
+  }
+
+  // NOTE buttons are mounted in order of tab precedence and arranged with CSS
+  // https://www.maxability.co.in/2016/06/13/tabindex-for-accessibility-good-bad-and-ugly/
+  return (
+    <DialogActions className={classes.dialogActions}>
+      <Button
+        variant='contained'
+        color='primary'
+        onClick={handleNext}
+        className={classes.button}
+        disabled={nextIsDisabled(activeStep, formFields)}
+      >
+        {getNextBtnLabel()}
+      </Button>
```

Comment:
We'll want to set the `type` for the dialog actions if the next button is set to submit, so screen readers/keyboard users can hit submit on enter (if the dialog has focus).

If that doesn't work as intended, we should consider wrapping the step content with a `<Form />`, setting an id on the form, and then additionally setting the prop `form` on the button to that id.